java/system-test/system-test-drivers/src/main/java/sleeper/systemtest/drivers/statestore/AwsStateStoreCommitterDriver.java

```diff
@@ -57,4 +68,43 @@ private void sendMessageBatch(List<StateStoreCommitMessage> batch) {
                                 .withMessageBody(message.getBody()))
                         .collect(toUnmodifiableList())));
     }
+
+    @Override
+    public void pauseReceivingMessages() {
```

Comment:
Naming pause and resume for method signature  but saying disabled and enabled in logs and action.
Do we want to align?

Reply:
I think that's a question for how directly we should couple the domain-specific language (DSL) to the implementation. The idea for the setup of these system tests is that we have a language in which we write the tests, which should not care about the implementation, and then we have drivers that interact with the implementation.

In abstract, a DSL doesn't care how it's implemented. If the test DSL is for buying books, you shouldn't know from reading a test whether it's operating against a physical book shop with a robot, a web UI, or a REST API. The test is written in implementation-agnostic language, but I think if you're reading the logs for a test execution, you'd expect to see either "entered the shop, went to aisle 1, looked on the top shelf", or "made a GET with search query 'My Book'".

I'm not sure what the right level of abstraction is here. I wouldn't want the language of the test, or the interface to the driver, to know that the way we pause/resume commits is by enabling/disabling the event source mapping. I want to be able to easily replace that detail if we find a better way to do it.